src/plugins/opencl_support/src/OpenCLSupportPlugin.cpp

```diff
@@ -36,8 +36,10 @@
 namespace U2 {
 
 extern "C" Q_DECL_EXPORT Plugin *U2_PLUGIN_INIT_FUNC() {
```

Comment:
If you want to remove the plugin from our distribution bundles why don't you just remove in create_bundle.sh/cmd ?